.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+version: 2.1
+commands:
+  python_build:
+    parameters:
+      python_version:
+        type: string
+        default: ""
+    steps:
+      - checkout
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python<< parameters.python_version >>/site-packages
+      - run: cat requirements/* > requirements.lock
```

Comment:
Would not `cat requirements/circleci-* > requirements.lock` be more accurate?

Reply:
Addressed in a302e9f.